src/cascadia/TerminalApp/App.cpp

```diff
@@ -1252,19 +1251,22 @@ namespace winrt::TerminalApp::implementation
     {

```

Comment:
do we still need two copies of GetIconFromProfile? This code is duplicated in Tab.cpp now.